index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./styles.css">
+  <title>Portfolio</title>
+
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@300;400;500;600;700;800&family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
+</head>
+<body>
+  
+<nav>
+  <a href="#" class="logo">John Doe</a>
+  <div id="menu-button"><img id="humberger" src="./Images/Menu humberger icon.svg"></div>
+</nav>
+
+<section id="first">
+  <div id="fs-content">
+    <h1 id="greeting">Hey there. I'm Alex</h1>
+    <h3>I am a Software Developer</h3>
+    <p>
+      I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+      don't hesitate to contact me.
+    </p>
+    <div id="soc-med">
+      <img src="./images/Github.svg" alt="GitHub">
+      <img src="./images/LinkedIn.svg" alt="LinkedIn">
+      <img src="./images/Peace-sign.svg" alt="Peace-sign">
+      <img src="./images/Twitter.svg" alt="Twitter">
+      <img src="./images/M.svg" alt="M">
+    </div>
```

Comment:
- [ ] Kindly make to wrap your social media links into the `ul` tag and wrap each icon in the `li` tag.
- [ ] I highly recommend wrapping each icon into an `a` or `button` tag in order to increase website accessibility.

Reply:
Good job following the advice of the previous reviewer